content/en/docs/tasks/access-application-cluster/list-all-running-container-images.md

```diff
@@ -23,7 +23,7 @@ In this exercise you will use kubectl to fetch all of the Pods
 running in a cluster, and format the output to pull out the list
 of Containers for each.
 
-## List all Containers in all namespaces
+## List all Container Images in all namespaces
```

Comment:
Hello @rajeshdeshpande02 ,
A couple of thoughts about changing the headings:
Right now, the headings refer generally to Container resources (not Container.image). Filtering on image field is a detail.
If you change the headings, the links in the page will be changed.
If you want to change the headings, I suggest you update the headings to use sentence case, for example:
List all Container images in all namespaces
List Containers by Pod (okay as I think the original author meant the Container object; image is the field in the Container resource)
What do you think?

Reply:
Hello @kbhawkey ,
Thanks for your review.
I have made changes for heading sentence case.
If you see title and description of this page, it refers to container images. Also, all commands discussed are listing container images only. So heading should also indicate that it is refering container images and not container itself.
Also, i checked referneces to this heading but didnt find any.